packages/reactotron-server/src/server/pluginManager/index.ts

```diff
@@ -0,0 +1,65 @@
+import * as path from "path"
```

Comment:
Let me start by saying that I am not sure if I like how this is laid out yet. I am throwing this out there as it is the first solution I landed on that works. Right now plugins can add schema, queries, subscriptions, etc to GraphQL. They can also provide event handlers for commands. These two things together can open up the ability to offload some functions from core like state snapshots. Why not put them in `reactotron-server-plugin-redux` or `reactotron-server-plugin-mst` since in core we don't really care. Not suggesting we dump subscription backups from core _yet_ but I want to open that door so we can slim core down to basically nothing _eventually_.